mzroll/mainwindow.cpp

```diff
@@ -3093,6 +2946,245 @@ QWidget* MainWindow::eicWidgetController() {
+
+		return btnAutoZoom;
+	}
+	else if (btnName == "btnShowTic") {
+
+		QToolButton *btnShowTic = new QToolButton(parent);
+		btnShowTic->setCheckable(true);
+		btnShowTic->setChecked(false);
+		btnShowTic->setIcon(QIcon(rsrcPath + "/tic.png"));
+		btnShowTic->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+		btnShowTic->setToolTip(tr("Show TICs"));
+		connect(btnShowTic, SIGNAL(toggled(bool)), mw->getEicWidget(), SLOT(replot()));
+		connect(btnShowTic, SIGNAL(toggled(bool)), mw->getEicWidget(),
+				SLOT(showTicLine(bool)));
+
+		return btnShowTic;
+
+	}
+	else if (btnName == "btnShowBarplot") {
+
+		QToolButton *btnShowBarplot = new QToolButton(parent);
+		btnShowBarplot->setIcon(QIcon(rsrcPath + "/barplot.png"));
+		btnShowBarplot->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+		btnShowBarplot->setToolTip(tr("Show Barplot"));
+		btnShowBarplot->setCheckable(true);
+		btnShowBarplot->setChecked(true);
+		connect(btnShowBarplot,SIGNAL(toggled(bool)),  mw->getEicWidget(), SLOT(showBarPlot(bool)));
+		connect(btnShowBarplot,SIGNAL(toggled(bool)), mw->getEicWidget(), SLOT(replot()));
+		return btnShowBarplot;
+
+	}
+	else if (btnName == "btnShowIsotopeplot") {
+
+		QToolButton *btnShowIsotopeplot = new QToolButton(parent);
+		btnShowIsotopeplot->setIcon(QIcon(rsrcPath + "/isotopeplot.png"));
+		btnShowIsotopeplot->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+		btnShowIsotopeplot->setToolTip(tr("Show Isotope Plot"));
+		btnShowIsotopeplot->setCheckable(true);
+		if (mw->isotopePlotsDockWidget->isVisible()) {
+			btnShowIsotopeplot->setChecked(true);
+		} else {
+			btnShowIsotopeplot->setChecked(false);
+		}
+
+		connect(btnShowIsotopeplot,SIGNAL(toggled(bool)),  mw->getEicWidget(), SLOT(showIsotopePlot(bool)));
+		connect(btnShowIsotopeplot,SIGNAL(toggled(bool)),  mw->getEicWidget(), SLOT(showIsotopicBarPlot()));
+		connect(btnShowIsotopeplot,SIGNAL(toggled(bool)), mw->isotopeWidget, SLOT(updateIsotopicBarplot()));
+
+		return btnShowIsotopeplot;
+
+	}
+	else if (btnName == "btnShowBoxplot") {
+		
+		QToolButton *btnShowBoxplot = new QToolButton(parent);
+		btnShowBoxplot->setIcon(QIcon(rsrcPath + "/boxplot.png"));
+		btnShowBoxplot->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+		btnShowBoxplot->setToolTip(tr("Show Boxplot"));
+		btnShowBoxplot->setCheckable(true);
+		btnShowBoxplot->setChecked(false);
+		
+		connect(btnShowBoxplot,SIGNAL(toggled(bool)),  mw->getEicWidget(), SLOT(showBoxPlot(bool)));
+		connect(btnShowBoxplot,SIGNAL(toggled(bool)), mw->getEicWidget(), SLOT(replot()));
+
+		return btnShowBoxplot;
+
+	}
+	else if (btnName == "btnShowSplines") {
+
+		QToolButton *btnShowSplines = new QToolButton(parent);
+		btnShowSplines->setIcon(QIcon(rsrcPath + "/splines.png"));
+		btnShowSplines->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+		btnShowSplines->setToolTip(tr("Show Splines"));
+		btnShowSplines->setCheckable(true);
+		btnShowSplines->setChecked(false);
+		btnShowSplines->setVisible(false);
```

Comment:
This button "btnShowSplines" was not in develop branch or might be the case it was not visible. But right now in this branch this button is added(or visible) in the end on toolbar of eicWidget.